src/change_stream.ts

```diff
@@ -411,6 +412,15 @@ export interface ChangeStreamCursorOptions extends AbstractCursorOptions {
   startAtOperationTime?: OperationTime;
   resumeAfter?: ResumeToken;
   startAfter?: boolean;
+  /**
+   * Comment to apply to the operation.
+   *
+   * In server versions pre-4.4, 'comment' must be string.  A server
+   * error will be thrown if any other type is provided.
+   *
+   * In server versions 4.4 and above, 'comment' can be any valid BSON type.
+   */
+  comment?: any;
```

Comment:
why are we adding this explicitly here if it already extends `AbstractCursorOptions` which has it defined?

Reply:
I'm not sure but the reason I'm pointing it out here is that this makes it inconsistent with most of the other places we have declared types, and it also defeats the purpose of "extends" in the way we've used it so far in all these interfaces (i.e., as the shared base, so that we don't have to copy paste exactly the same description in 50 different files). If we're going to declare all properties on all types individually, I'd rather we just have separate types. But I think we could also have a documentation solution - i.e., have something that follows the tree and compiles the complete types for doc purposes. Either way, the less consistency we have in how we do things now, the more headache it will be to change everything to be uniform in one way or another later on, so my vote is to keep our convention.

Reply:
I removed the redundant type declarations.